app/src/components/sections/ContractManagementView/AdvertiseContractSection.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="flex flex-col gap-6">
+    <span v-if="teamIsFetching" class="loading loading-spinner loading-lg"></span>
+    <div v-if="!teamIsFetching && team" class="flex flex-col gap-5 w-full items-center">
+      <CardComponent class="w-full" title="Advertise Contract">
+        <template #card-action>
+          <div>
+            <ButtonUI
+              variant="primary"
+              :enabled="team.ownerAddress == useUserDataStore().address"
+              data-test="createAddCampaign"
```

Comment:
It's a bad patter to use useUserDataStore().address

You have to call the store like team, then call the store.address
This is not efficient, every time the component reload, he will call useUserDataStore() function to instanciate the store. But you don't need to instanciate the store every time the component reload.